### ISSUE TITLE
DC-1 not working in Live 9.2.2 on OSX 10.10.4

### ISSUE THREAD

**Issue by lkjts** [mpiraux/CMD-DC-1-Ableton-9-Control-Surface-Script#5]
[Log.txt](https://github.com/mpiraux/CMD-DC-1-Ableton-9-Control-Surface-Script/files/149566/Log.txt)

Hello, first I'd like to say big thanks for the script. I got brand new DC-1 in an auction very cheap and soon I was very disappointed, first with discovering the lack of velocity sensitive pads (helooo Behringer, I mean, WTF?) Luckily i found your script! 

I managed to get the DC-1 to show up in control surface list, and i did get the square around the clips. But it seems I'm not getting anything on the DC-1 to be able to actually control Live, I just get MIDI notes coming into Live. The DC does not control any Live's parameters, not do the buttons and pads change colour to indicate anything being controlled.

I would very much appreciate some help!
Thanks,

**Comment by mpiraux** [comment]
Hello,

I didn't found anything suspicious in your Log.txt, the script is initializing just fine. My guess would be that your DC-1 is not on the default channel. Could you make sure that it is using the 5th MIDI channel (which is numbered 6 in the Behringer channel changer software) ?

**Comment by lkjts** [comment]
Hi, thanks for looking into my issue. Yes, this seems to do the trick. I had no idea I have to do this to work, it doesn't say anything about it in the DC-1 manual, nor did I found anything online. After I read you reply I went to Behringer site and saw the MIDI channel change software there.

All in allI am still confused with lack of official support for this device, and most of all the lack of MIDI velocity on the pads. I will probably sell this unit whilst it is still new.
